src/com/rcv/GuiConfigController.java

```diff
@@ -48,16 +77,41 @@ public void initialize(URL location, ResourceBundle resources) {
 
     choiceOvervoteRule.getItems().addAll(Tabulator.OvervoteRule.values());
     choiceOvervoteRule.getItems().remove(Tabulator.OvervoteRule.RULE_UNKNOWN);
-    choiceOvervoteRule.getSelectionModel().select(0);
+
+    datePickerContestDate.setConverter(
+        new StringConverter<>() {
+          final DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+
+          @Override
+          public String toString(LocalDate date) {
+            return (date != null) ? dateFormatter.format(date) : "";
```

Comment:
Style nit: I'm not in the habit of putting the ternary condition in parentheses because it feels superfluous. Does the Google style say that we should? Any particular arguments for or against?

Reply:
That's my own peculiarity (I find it to be more readable). But sure, I'll remove the parens and won't use them in the future.